src/main/java/sirius/biz/storage/layer2/jdbc/SQLBlobStorageSpace.java

```diff
@@ -11,6 +11,7 @@
 import sirius.biz.storage.layer2.BasicBlobStorageSpace;
 import sirius.biz.storage.layer2.Blob;
 import sirius.biz.storage.layer2.Directory;
+import sirius.biz.storage.layer2.mongo.MongoBlob;
 import sirius.biz.storage.layer2.variants.BlobVariant;
```

Comment:
```suggestion
import sirius.biz.storage.layer2.Directory;
import sirius.biz.storage.layer2.variants.BlobVariant;
```
Removes unused import if the other comment in the same file is committed.